src/queries/catalog/location-query/location-query-executor.ts

```diff
@@ -0,0 +1,17 @@
+import type { BasicQueryData } from '../../query';
+import type { LocationQuery, LocationQueryResult } from './location-query-format';
+import { labelSourceRange } from './location-query-format';
+
+
+export function executeLocationQuery({ ast }: BasicQueryData, queries: readonly LocationQuery[]): LocationQueryResult {
+	return {
+		'.meta': {
+			/* there is no sense in measuring a get */
+			timing: 0
```

Comment:
given there are multiple hashmap access operations, i would in-fact measure the time, as we are no longer constant!